base/system/autochk/autochk.c

```diff
@@ -180,8 +180,8 @@ WaitForKeyboard(VOID)
                         NULL);
     if (Status == STATUS_PENDING)
     {
-        /* Wait 3s */
-        Timeout.QuadPart = (LONG)-3*1000*1000*10;
+        /* Wait 10s */
+        Timeout.QuadPart = (LONG)-10*1000*1000*10;
```

Comment:
This code was written three days ago, and it appears to be a conscious decision by the author to use a 3 second timeout. So a change should be preceded by a discussion with @HeisSpiter about why you think 3 seconds is not a good choice.

Reply:
I changed it because it doesn't feel like someone who is seeing this screen for the first time, or someone who is slower for what ever reason, has enough time to process what's going on and decide if they want to skip.

A user friendliness and accessibility change.

Reply:
Then please convince Pierre. Just rewriting recent code is considered rude.

Reply:
My apologies. Did not intend to be rude. This is the second time I've contributed to a project of this scale, so I haven't quite picked up on proper etiquette yet.

@HeisSpiter Is this change alright with you given my reasons above or would you like me to revert it?